tutorials/QML_with_Classiq/QML_with_Classiq.ipynb

```diff
@@ -0,0 +1,1203 @@
+{
```

Comment:
<p>Same comment as before on the naming because the user might overlook it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Reply:
<p>It's an issue of the ReviwNB rendering, The naming is indeed different. </p>